Dockerfile

```diff
@@ -22,4 +22,6 @@ WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 COPY --from=publish /app/publish .
+ARG version=unknown
+RUN echo $version > /app/wwwroot/version.txt
```

Comment:
Oh, it seems that `wwwroot` is not the right place in your project: 

![image](https://user-images.githubusercontent.com/1157864/74974718-84a17e80-5404-11ea-9550-c37bb637f9c3.png)

Reply:
Done, I removed the subfolder `wwwroot` of the path